benchmarks/lineformat.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
```

Comment:
Perhaps we can rename this file from `benchmarks/lineformat.py`  to `benchmarks/lineprotocol.py` as a way to have its name be a bit more self describing